### DISCUSSION TITLE
Python support(Interact)

### DISCUSSION THREAD

**Discussion by Eudritch** [rocksdanister/lively#562] [Ideas]
I'm a big fan of lively wallpaper and was wondering if there was a way for Python to interact with it. I'm a programmer myself, but I can't seem to find a way to use Python to set HTML as the Windows 10 wallpaper. Please get back to me. and I'd like to contribute to the development of Python 3 support.

**Comment by rocksdanister** [comment]
https://github.com/rocksdanister/lively/wiki/Command-Line-Controls
